qiita_db/metadata_template.py

```diff
@@ -1181,9 +1181,8 @@ def create(cls, md_template, study):
         ----------
         md_template : DataFrame
             The metadata template file contents indexed by samples Ids
-        study : Study or RawData
-            The study to which the metadata template belongs to. Study in case
-            of SampleTemplate and RawData in case of PrepTemplate
+        study : Study
+            The study to which the sample template belongs to.
```

Comment:
I think this is an accurate change, but please correct me if I'm wrong, and I can revert